containers/ws/label-install

```diff
@@ -44,7 +44,7 @@ chown root:root /host/etc/cockpit/ws-certs.d /host/etc/cockpit/machines.d
 mkdir -p /etc/ssh
 
 # For podman, generate a systemd unit for starting on boot
-if [ "${container:-}" = oci ] && [ -n "$IMAGE" ] && [ ! -e /host/etc/systemd/system/cockpit.service ]; then
+if [ "${container:-}" = oci -o "${container:-}" = podman ] && [ -n "$IMAGE" ] && [ ! -e /host/etc/systemd/system/cockpit.service ]; then
```

Comment:
The old code uses `&&` for logic, should the new code maybe use `||`?

Reply:
I prefer `||` as well, but we would need to use some () grouping then, which is syntactically awkard (or maybe even impossible -- this might then need some nested `if`s rather?)